pgbouncer/check.py

```diff
@@ -109,9 +111,13 @@ def _collect_stats(self, db, instance_tags):
                         continue
 
                     desc = scope['descriptors']
+
+                    # Some versions of pgbouncer have extra fields at the end of SHOW POOLS
                     if len(row) == len(cols) + len(desc) + 1:
-                        # Some versions of pgbouncer have an extra field at the end of show pools
                         row = row[:-1]
+                    elif len(row) == len(cols) + len(desc) + 2:
```

Comment:
As we discussed in person, we cannot be 100% sure if we'll get all extra columns in other versions, but since we can't really get the column titles without resorting to some crazy VIEW trickery (http://peter.eisentraut.org/blog/2015/03/25/retrieving-pgbouncer-statistics-via-dblink/) - this sensible assumption is probably the best we can do. Tests seem to confirm this now works on newer pgbouncers.